src/components/languageSelect/index.js

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react"
+
+import i18n from "../../i18n"
+import Menu from "../menu"
+
+import "./style.css"
+
+const LanguageSelect = () => {
+  const [selectedLanguage, setSelectedLanguage] = useState("en")
```

Comment:
Could we use a const for the default language. We could also try and load a desired default language via the environment variables